lib/faktory_worker/push_pipeline/acknowledger.ex

```diff
@@ -1,11 +1,16 @@
 defmodule FaktoryWorker.PushPipeline.Acknowledger do
   @moduledoc false
 
+  alias FaktoryWorker.Job
+
   @behaviour Broadway.Acknowledger
 
   @impl true
-  def ack(_ack_ref, _successful_messages, _failed_messages) do
-    # need to work out what this should do
+  def ack(_ack_ref, _successful_messages, failed_messages) do
+    Enum.each(failed_messages, fn %{data: {pipeline, job}} ->
```

Comment:
I wonder if `job` should be called `payload` to be consistent with `FaktoryWorker.Job.perform_async`.

Reply:
Yeah that makes sense, I will update.